packages/terra-functional-testing/lib/config/wdio.conf.js

```diff
@@ -113,9 +125,15 @@ exports.config = {
   // your test setup with almost no effort. Unlike plugins, they don't add new
   // commands. Instead, they hook themselves up into the test process.
   services: [
+    [TerraService],
+    [AssetService, {
+      ...SITE && { site: SITE },
+      ...LOCALE && { locale: LOCALE },
+      ...THEME && { theme: THEME },
+      ...fs.existsSync(defaultWebpackPath) && { config: defaultWebpackPath },
```

Comment:
how will consumers override this value? Will they need to re-define all the services? or remove this service & re-define it?

Reply:
They'd need to remove this service and redefine it. We could potentially expose an environment variable now that it represents a relative path.

This is a continuation of the discussion [here](https://github.com/cerner/terra-toolkit/pull/412#discussion_r438309783). The strategy for merging/overriding configurations isn't ideal, but it is something consumers will likely need to do from time to time. The second option in the linked thread might make it slightly easier for overriding options, but isn't ideal either.

Reply:
> They'd need to remove this service and redefine it.

This doesn't seem like a great option for our consumers..And this will lock us into the service order. I wouldn't expect it to change, but if for some reason it did, we'd need to maintain the same order for passivity.

Reply:
Do you have any ideas for making this easier? This does not lock us into a service order, even with the current API. Consumers shouldn't be dice rolling the index of a service. They can read the service name from the array.

A couple ideas mentioned above that could help mitigate this would be the environment variables or the keyed configuration value for each service. I don't think either of these fully resolve the issue when merging webpack configuration, but they could help.

Reply:
I think its fine to expect wepback config to be defined at root.